### PR TITLE
docs: add Chinese translations

### LINKED ISSUE #340: I want to provide a Chinese README file, is it feasible?
### Describe the feature

Maybe the translation is not good enough, but I have thought and weighed every sentence, and try to use the appropriate Chinese expression.

### Additional information

- [x] Would you be willing to help implement this feature?

### PR DESCRIPTION
**Pull request by AllenYu0118**
### 🔗 Linked issue
#340 

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Close #340 

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+```ts
+await ofetch('/url', { ignoreResponseError: true })
+```
+
+## ✔️ 自动重试
+
+如果发生错误或者响应状态码符合`重试状态码`条件时，`ofetch` 会自动重新发送请求。
+
+**重试状态码:**
+- `408` - 请求超市
```
Comment (BabyLy233) [inline README.zh-CN.md:115]: 请求超时

**Review by BabyLy233 — COMMENTED** [review]
fix typo

File: README.zh-CN.md (hunk 1 above)
Comment (AllenYu0118) [reply README.zh-CN.md]: thanks🤩

**Comment by antfu** [comment]
I am not so sure if we want to maintain translations for docs, given the fact that:

- We duplicate the sources of truth. Every time we update something, we need to spend extra effort updating translations or we create misalignment
- `ofetch` is not a very user-facing library I suppose?
- AI nowadays could provide very accurate translations already

**Comment by pi0** [comment]
Thanks for PR dear @AllenYu0118 and willing to help on translations. I think @antfu is correct as keeping the docs up to date can be challenging in the future. We might think of automated translations in website in the future. I have moved the follow to this thread: https://github.com/unjs/website/issues/207

**Comment by AllenYu0118** [comment]
I agree with your idea,  thank for your response.🤔

**Pull request closed without merging**